rust-toolchain

```diff
@@ -1 +1,3 @@
-1.56.1
+[toolchain]
```

Comment:
can you revert these changes?

I've wanted to move the toolchain over to the `rust-toolchain.toml` format for a while but CI uses `actions-rs/toolchain` and this GitHub Action does not support the toml version of the toolchain file.

Reply:
The CI failures are because of the changes to the toolchain file.

Reply:
reverted